projects/core/styles/mixins/wrapper.scss

```diff
@@ -1,5 +1,7 @@
+@import '../variables/media.scss';
+
 @mixin wrapper-hover {
-    @media (hover: hover) {
+    @media ($tui-mobile) {
```

Comment:
Unnecessary brackets here and in some other places

Reply:
But

<img width="200" alt="image" src="https://github.com/user-attachments/assets/ec9f54fa-6652-4363-a0a1-b803436c54bb">

<img width="414" alt="image" src="https://github.com/user-attachments/assets/ab85f3bd-cd30-4bc6-91ca-e42017b5da9f">